src/github/communitystats-mapping.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable, BadRequestException } from '@nestjs/common';
+
+@Injectable()
+export class CommunitystatsMappingService {
+  private eventMap = {
```

Comment:
Can we hardcode the Mapping for the Events, or is there a way to do that dynamically?

Reply:
I think hardcode is probably best, these are not going to change.